spec/features/admin_can_view_dashboard_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.feature "Admin visits dashboard" do
+  scenario "admin sees dashboard" do
+
+    User.create(first_name: "Bon", last_name: "Jovi", address: "123 crazy street", email: "deadoralive@awesome.com", username: "bonjovirules", password: "deadoralive", role: "admin")
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+
+    visit admin_dashboard_path
+
+    expect(page).to have_content("Admin Dashboard")
+  end
+
+  scenario "admin visits dashbaord from root" do
+    item_1 = category.items.create!(title: "Mask", description: "This is for your face", price: 20.00, image: "https://slack-imgs.com/?c=1&url=http%3A%2F%2Fwww.scubadivingdreams.com%2Fwp-content%2Fuploads%2F2015%2F11%2Fthe-best-scuba-snorkel-mask-mares-i3-sunrise.jpg")
+    item_2 = category.items.create!(title: "Fin", description: "This is for your face", price: 10.00, image: "https://slack-imgs.com/?c=1&url=http%3A%2F%2Fwww.scubadivingdreams.com%2Fwp-content%2Fuploads%2F2015%2F11%2Fthe-best-scuba-snorkel-mask-mares-i3-sunrise.jpg")
+    item_3 = category.items.create!(title: "Sunscreen", description: "This is for your face", price: 6.00, image: "https://slack-imgs.com/?c=1&url=http%3A%2F%2Fwww.scubadivingdreams.com%2Fwp-content%2Fuploads%2F2015%2F11%2Fthe-best-scuba-snorkel-mask-mares-i3-sunrise.jpg")
+
+    User.create(first_name: "Bon", last_name: "Jovi", address: "123 crazy street", email: "deadoralive@awesome.com", username: "bonjovirules", password: "deadoralive", role: "admin")
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+
+    visit root_path
+
+    click_link "Dashboard"
+
+    expect(page).to have_content("Admin Dashboard")
+  end
+end
```

Comment:
It looks like we need to remember to put a link to the dashboard either on the navbar or the user showpage if they are an admin.

Reply:
ya i figured there should be some sort of way an admin can get to the dashboard without manually typing it in